content/topics/solo-learn/python/intro-to-python/2-going-deeper-with-python/index.md

```diff
@@ -0,0 +1,19 @@
+---
+content_type: topic
+flavours:
+- none
+prerequisites:
+  hard:
+  - topics/solo-learn/python/intro-to-python/1-getting-started-with-python
```

Comment:
Change this to a soft pre-req or they wont be able to start this until we move the pre-req to complete